### ISSUE TITLE
Hi, is this game engine dead?

### ISSUE THREAD

**Issue by Raj2032** [amethyst/bracket-lib#261]
Hi, is this game engine dead?

**Comment by thebracket** [comment]
No, I'm on a long paternity leave. The api is pretty much frozen for
compatibility with the roguelike tutorial and Hands-on Rust, but I have
fixes in the works for when I return.

**Comment by Raj2032** [comment]
ah right I understand, thanks :)

**Comment by john-science** [comment]
@thebracket If you would like a little help keeping these projects going, I could help with tickets.  In particular, I could probably be quite helpful cleaning up the `rustrogueliketutorial`.   Since that's just documentation, it would probably be less worrisome to you that some rando was helping on the project.

Just offering!  There are a lot of low-hanging-fruit tickets over there.

**Comment by thebracket** [comment]
I may well take you up on that, I'd been considering putting up a "help wanted". We have to keep the goal of "Hands-on Rust code still compiles/runs" (that book has kept my lights on!) - otherwise, I could definitely use some help at this point. I hadn't expect to adopt a little girl with some serious sleep issues (she literally has us up between 1am and 3am every single night) - which has made keeping up with everything physically really hard. :-(

**Comment by Zireael07** [comment]
Help wanted tags are very common on many repos and would likely reduce the load on you!

**Comment by Tom-CSTech** [comment]
I have to wonder if there were not active contributors before to whom you could have given elevated privileges to keep the project moving forward, because the appearance of being dead can surely scare many people away and often the inertia may never return. For using a collaborative platform like this, and having had a respectable project with such a following, it seems a waste not to delegate a little more. In any case, as far as your own contributions, I appreciate the focus that your current life takes (I am not ready to raise a child yet...), and I wish you all the best!
